bin/blockfrostUtils.js

```diff
@@ -0,0 +1,74 @@
+
+import { readFile } from 'fs/promises';
+import { BlockFrostAPI } from "@blockfrost/blockfrost-js";
+
+/**
+ * 
+ * @param {*} blockfrostKey 
+ * @param {*} walletAddress 
+ * @returns 
+ */
+const queryUTxOviaBlockfrost = async (blockfrostKey, walletAddress) => {
+  const blockfrost = new BlockFrostAPI({
+    projectId: blockfrostKey,
+  });
+  const utxos = await blockfrost.addressesUtxosAll(walletAddress);
+  let TxHash = "";
+  let TxIx = 0;
+  let Amount = 0;
+  for (const utxo of utxos) {
+    if (utxo.amount.reduce((sum, item) => sum + parseInt(item.quantity, 10), 0) > 1000000) {
+      TxHash = utxo.tx_hash; 
+      TxIx = utxo.output_index;
+      Amount = utxo.amount.reduce((sum, item) => sum + parseInt(item.quantity, 10), 0) 
+    }
+  }
+  return { TxHash, TxIx, Amount };
+}
+
+/**
+ * 
+ * @param {blockfrostKey for the transaction. Beware that blockfrost key is different for different testnets and mainnet} blockfrostKey 
```

Comment:
Same here with JSDocs, follow https://jsdoc.app/howto-es2015-classes

It should look like:
`@param {string} blockfrostKey - Key for the transaction. Beware that blockfrost key is different for different testnets and mainnet`